css/properties/transition-property.json

```diff
@@ -144,7 +144,7 @@
               },
               {
                 "prefix": "-webkit-",
-                "version_added": true
+                "version_added": "2"
```

Comment:
Here you've set samsunginternet_android to 2 instead of safari to 3.1 and safari_ios to 2.

Reply:
Whoops -- let's just pretend that didn't happen and move along, yeah?  😝